util/cosmos/iscn/query.js

```diff
@@ -0,0 +1,203 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Tendermint34Client } from '@cosmjs/tendermint-rpc';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { decodeTxRaw } from '@cosmjs/proto-signing';
+import { QueryClient, StargateClient } from '@cosmjs/stargate';
+import { MsgCreateIscnRecord } from '@likecoin/iscn-message-types/dist/iscn/tx';
+
+import setupISCNExtension from './iscnQueryExtension';
+import { timeout } from '@/util/misc';
+
+const ISCN_RPC_URL = '/api/cosmos/rpc';
```

Comment:
need to add:
import { EXTERNAL_URL } from '@/constant';
const ISCN_RPC_URL = `${EXTERNAL_URL}/api/cosmos/rpc`;